modelscan/cli.py

```diff
@@ -69,6 +74,20 @@ def cli() -> None:
     type=click.Path(exists=True, dir_okay=False),
     help="Specify a settings file to use for the scan. Defaults to ./modelscan-settings.toml.",
 )
+@click.option(
+    "-f",
+    "--format",
+    type=click.Choice(["console", "json", "custom"]),
+    default="console",
+    help="Format of the output. Options are console, json, or custom (to be defined in settings-file). Default is console.",
+)
```

Comment:
For now, let's avoid the additional CLI flag here. The report type can be set in the settings file and we can work on the cli usability if it's requested in another PR